scripts/generate_sphinx_docs.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+BUILD_FROM_BRANCH="${1:-master}"
```

Comment:
Can you add comments here regarding what's happening on the various lines? Also please ensure it adheres to the Google style guide for [Bash](https://google.github.io/styleguide/shell.xml) if it doesn't.